app/src/main/java/com/github/se/polyfit/ui/components/ingredients/IngredientNutritionEditFields.kt

```diff
@@ -109,3 +105,34 @@ fun IngredientNutritionEditFields(
     }
   }
 }
+
+private fun tryCatchBlock(
```

Comment:
```suggestion
private fun formatText(
```
The name of the function doesn't give information about what the function does.

Reply:
Yes you're right it was just changed quickly without paying attention to the name, I'll modify it and make sure I don't repeat this mistake.